run_action.py

```diff
@@ -174,19 +174,40 @@ def main():
 
     github_token = os.environ.get("INPUT_GITHUB_TOKEN")
     github_api_url = os.environ.get("GITHUB_API_URL")
+    
     request_url = "%s/repos/%s/issues/%s/comments" % (
         github_api_url,
         repo,
         args.pull_request_id,
     )
 
+    headers = {
+        "Authorization": "token %s" % github_token,
+    }
+
+    one_comment = os.environ.get("INPUT_ONE_COMMENT")
+    if one_comment.lower() in ("true", "1"):
+        ## Search comments
+        requset_url_list_comments = requests.get(request_url,
```

Comment:
There's a typo in `requset`, but what about this instead:

```suggestion
        pr_comments = requests.get(request_url,
```

(Need to change it below as well)